test/ngSanitize/sanitizeSpec.js

```diff
@@ -261,8 +261,8 @@ describe('HTML', function() {
       });
     });
 
-    if (!/Edge\/(16|17)/.test(window.navigator.userAgent)) {
-      // Skip test on Edge 16 due to browser bug.
+    if (!/Edge\//.test(window.navigator.userAgent)) {
```

Comment:
Could add `\d` or `\d{2,}` to make it a little closer to the previous? 🤷‍♂️

Reply:
You can now make such minor code suggestions directly from GitHub. :) The icon with sheets of paper & plus/minus on the front one.
It looks like that:
```suggestion
    if (!/Edge\/\d{2,}/.test(window.navigator.userAgent)) {
```

Reply:
And then I can apply the suggestion with one button.

Reply:
Although I heard it messes with the commit ownership?

Reply:
As long as you don't use the github merge button for squashing, I think it is OK.